autoscheduler/frontend/src/tests/Meeting.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import * as React from 'react';
+
+import MeetingCard from '../components/MeetingCard';
+import { Meeting, MeetingType, Section } from '../types';
```

Comment:
Does this not throw an error for you? I get a `Cannot find module '../types'` error when I try to run the tests.

Reply:
It does, I'll have to refactor it to reflect the latest commits to frontend/master